kill.go

```diff
@@ -58,10 +59,18 @@ signal to the init process of the "ubuntu01" container:
 func parseSignal(rawSignal string) (syscall.Signal, error) {
 	s, err := strconv.Atoi(rawSignal)
 	if err == nil {
-		return syscall.Signal(s), nil
+		signal := syscall.Signal(s)
+		if unix.SignalName(signal) == "" {
```

Comment:
While we are unlikely to see many new signals being defined, I don't see a reason to deny sending a signal if the kernel allows you to do it.

Reply:
Would a log entry be useful (to catch a potentially incorrect signal?)

Reply:
no, send the signal and let the kernel return an error, if  there even is one

Reply:
@crosbymichael see https://github.com/containerd/containerd/pull/4115 :)

Reply:
This is why I put this code in a separate commit (now removed).

@cyphar fixed

Reply:
Thanks.